packs/hpe-icsp/actions/icsp_mid_get.yaml

```diff
@@ -0,0 +1,23 @@
+---
+  name: "icsp_mid_get"
+  runner_type: "run-python"
+  description: "Retreive MID for a given server"
```

Comment:
It appears this action returns a list and allows you to retrieve information for multiple servers.

If that's the case it would be good to update the description to reflect that :)